doc/meetings/15.03.2023_gamification.md

```diff
@@ -0,0 +1,56 @@
+# Meeting 2023-03-15 Focus Group Gamification
+
+_Protocolled by: Markus_
+
+## Attendees:
+
+-   Markus
+-   Yvonne
+-   Pavlo
+-   Thorben
+
+## Agenda:
+
+- Goals of the game(s)
+- Related work
+- Use cases
+- Brain storming for interface
+
+## Meeting notes:
+
+- Goals of the game(s)
+   - class room (learning) +1
```

Comment:
We decided later that class room and citizen science will have lower priority, so they will be done after the use cases directly affecting the UX of PermaplanT (rather adding new features to PermaplanT)